cmake/posix/px4_impl_posix.cmake

```diff
@@ -267,22 +267,32 @@ elseif ("${BOARD}" STREQUAL "excelsior")
 		--sysroot=${HEXAGON_ARM_SYSROOT}/lib32-apq8096  -mfloat-abi=softfp -mfpu=neon -mthumb-interwork
 
 		)
-elseif ("${BOARD}" STREQUAL "rpi" AND "$ENV{RPI_USE_CLANG}" STREQUAL "1")
+elseif ("${BOARD}" STREQUAL "rpi")
+	SET(RPI_COMPILE_FLAGS
```

Comment:
Shouldn't this live in the toolchain file or does the existing structure force this?